neqo-crypto/src/ext.rs

```diff
@@ -99,7 +102,8 @@ impl ExtensionTracker {
         arg: *mut c_void,
     ) -> SECStatus {
         let d = std::slice::from_raw_parts(data, len as usize);
```

Comment:
Are you sure that this 'as' isn't being suppressed here also?

Reply:
It is not, because len is a c_uint (u32) so it's always lossless (even on 32 bits platform).
However since the wrap_handler_call is an other block, I have been able to move the allow(clippy) statement to wrap it only, which is always better \o/